docs/index_new.html

```diff
@@ -82,27 +82,27 @@ <h1 class="fade">Automate your team's development environments</h1>
     <div class="u-headline">Closing copy blah blah blah</div>
```

Comment:
There was no proper copy given to me for this. That being said, I will flag down Devrim for copy.